<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>801</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>99999</width>
    <height>99999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>File select</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="regionLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Region:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>761</width>
      <height>521</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>761</width>
      <height>521</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>761</width>
      <height>521</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Region</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NPG</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Birth Rate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Death Rate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>GDW</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Urbanization</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="columnNumberLineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Column number:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Median:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minValueLineEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxValueLineEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="medValueLineEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yearLineEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>380</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Average</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="averageLineEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>500</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dispValueLineEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>530</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>40</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>391</width>
      <height>421</height>
     </rect>
    </property>
    <widget class="QLabel" name="x_axis_label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>470</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>year</string>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_axis">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>351</width>
       <height>191</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="graphColumnLineEdit_1">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Regions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Column:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="graphRegionLineEdit_1">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>40</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
